test/unittest/Database/DatabaseLoadTests.cpp

```diff
@@ -580,10 +581,11 @@ void check_is_string_int(
         DatabaseDump& dump,
         DatabaseDump& container)
 {
+    // on windows json implementation "34.12" and "-76.98" are valid string int values
     std::vector<DatabaseDump> wrongValues = {DatabaseDump::object({{"one", 1}, {"two", 2}}), DatabaseDump::array({1, 2,
                                                                                                                   3,
                                                                                                                   4}),
-                                             true, 1234, 34.12, -9876, -76.98, "qwerty", "34.12", "-76.98"};
+                                             true, 1234, 34.12, -9876, -76.98, "qwerty", /*"34.12", "-76.98"*/};
```

Comment:
The invalidation of these values is not done by the Json library, but by our code in `string_to_int` and `string_to_uint`, as we are specifically checking the validity of the characters.

Please remove the comment and restore the removed values.

The same for the other cases in this same tests